Cargo.toml

```diff
@@ -8,9 +8,9 @@ rust-version = "1.66"
 
 [dependencies]
 rand = "0.8.5"
-lambdaworks-math = { git = "https://github.com/lambdaclass/lambdaworks", rev = "dfd0066" }
-lambdaworks-crypto = { git = "https://github.com/lambdaclass/lambdaworks", rev = "dfd0066" }
-lambdaworks-fft = { git = "https://github.com/lambdaclass/lambdaworks", rev = "dfd0066" }
+lambdaworks-math = { git = "https://github.com/lambdaclass/lambdaworks", rev = "ca0c68a" }
+lambdaworks-crypto = { git = "https://github.com/lambdaclass/lambdaworks", rev = "ca0c68a" }
+lambdaworks-fft = { git = "https://github.com/lambdaclass/lambdaworks", rev = "ca0c68a" }
```

Comment:
shouldn't these point to "133ffb3"?

Reply:
Yes, it can point to the latest one, I updated it to the one I needed

Reply:
Changed it

Reply:
Bumping versions breaks everything, I'll revert, we can update the version with the merkle tree update